Makefile

```diff
@@ -614,6 +614,11 @@ $(eval $(call add_define,WARMBOOT_ENABLE_DCACHE_EARLY))
 $(eval $(call add_define,BL2_AT_EL3))
 $(eval $(call add_define,BL2_IN_XIP_MEM))
 
+# Define the PRELOADED_BL32_BASE flag only if it is provided.
+ifdef PRELOADED_BL32_BASE
```

Comment:
I dont think PRELOADED_BL32_BASE build option is required generically. See my later comment regarding this.